### DISCUSSION TITLE
Is there an option to filter a field based on another field of the same item?

### DISCUSSION THREAD

**Discussion by abdonrd** [directus/directus#11748] [Archive]
With a `Matches` model like this where we have `participants` & `winners` (both relationships with `Profiles` model):

<img width="857" alt="Screenshot 2022-02-20 at 12 48 04" src="https://user-images.githubusercontent.com/1007051/154841029-62190f77-edbd-4457-b1e3-12855c3810a0.png">

We want to filter the `winners` field to show only profiles that are already defined as `participants`. Is there an option to filter a field based on another field of the same element?

<img width="860" alt="Screenshot 2022-02-20 at 12 54 35" src="https://user-images.githubusercontent.com/1007051/154841231-fb01147a-eaf6-4799-b506-95825cae7e8d.png">

**Comment by joselcvarela** [comment]
As far as I can tell, there's no way to do so.
We already have a feature request for computed fields, which would allow to read other fields from the same collection.

In the meantime you can create a relation to achieve your behaviour and use the filter rules

**Reply by abdonrd** [reply]
Thanks for the answer, @joselcvarela!

---

> In the meantime you can create a relation to achieve your behaviour and use the filter rules

What do you mean about this?

**Reply by joselcvarela** [reply]
For example, you could create a "winners" collection which is a M2M relating participants and matches. This way you could use Filter Rules
